change/@fluentui-react-conformance-6c13ae37-25da-4e48-9c39-0138a31c73f5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This should be `minor` (this package is on `0.x`) as it's a code change, correct?